conjur/resource.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+"""
+Resource module
+"""
+
+# pylint: disable=too-few-public-methods
+class Resource:
+    """
+    Used for representing Conjur resources
+    """
+    def __init__(self, **arg_params):
+        self.resource_type = arg_params['resource_type']
```

Comment:
`resource` prefix is redundant.
You already know it a resource.
so only `type` and `name` is enough, clear and concise.

Reply:
type is a reserved name in Python apparently

Reply:
so the param name can be resource_type, but the field can be `type`, so you could write `resource.type`

Reply:
Better options suggested here: https://stackoverflow.com/questions/2417979/can-i-use-the-variable-name-type-as-function-argument-in-python

I would go with `typ` or `type_`, and only for arg name.
The field can and should stay `type`.

Reply:
oh thought it was best practice for them to be called the same name so it would be `self.type = type`  or `self.res_type = res_type`

changing to `self.type = type_` and `self.name=name`